types/evidence.go

```diff
@@ -55,6 +55,7 @@ func (err *ErrEvidenceOverflow) Error() string {
 type Evidence interface {
 	Height() int64                                     // height of the equivocation
 	Address() []byte                                   // address of the equivocating validator
+	Bytes() []byte                                     // bytes which compromise the evidence
```

Comment:
`compromise -> comprise` ... I hope this method doesn't compromise the evidence :P

Reply:
LOL